packages/layer-composer/src/generators/types.ts

```diff
@@ -232,6 +234,9 @@ export interface TrackGeneratorConfig extends GeneratorConfig {
 export interface VesselEventsGeneratorConfig extends GeneratorConfig {
   type: Type.VesselEvents
   data: RawEvent[]
+  color?: string
+  track?: any
+  showTrackSegments?: boolean
```

Comment:
As far as I can see this is never practically set to false. I would remove configurability as long as it's not needed.

Reply:
Indeed, this is going to be removed